optimize/self-managed/optimize-deployment/configuration/system-configuration.md

```diff
@@ -222,6 +222,10 @@ Settings for webhooks which can receive custom alert notifications. You can conf
 
 Settings for automatic cleanup of historic process/decision instances based on their end time.
 
+:::note
+Camunda Platform 8 does not have external event or decision data yet. The process history cleanup runs but does not do anything at this time.
```

Comment:
I think there was maybe some confusion here in previous discussion, so will clarify here. There are four types of history cleanup, listed below:

1. Process data cleanup
2. Decision data cleanup
3. External event cleanup
4. External variable cleanup

And the behaviour for each mode is as follows:

**C7:**
- By default, all 4 cleanups are disabled
- Each can be enabled individually by config, and the cleanup is applied accordingly

**C8:**
- By default, all 4 cleanups are disabled
- Each can be enabled individually by config, although only process (1) and external variable (4) cleanup actually does anything, because decision data and external event data doesn't exist in C8

Does that make things clearer? With this in mind, I don't think the second part of this note is true, and I don't think that anything more than the first part is worth mentioning

Reply:
Clear as mud 😆 just kidding, this makes much more sense. So let's keep the first part only here.

```suggestion
Camunda Platform 8 does not have external event or decision data yet. 
```